adapter_pipelines/smart_seq2/adapter_example_static_demo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "WrapperSs2RsemSingleSample.gtf": "gs://broad-dsde-mint-dev-teststorage/demo/gencodev19_chr21.gtf",
+  "WrapperSs2RsemSingleSample.ref_fasta": "gs://broad-dsde-mint-dev-teststorage/demo/chr21.fa",
+  "WrapperSs2RsemSingleSample.ref_flat": "gs://broad-dsde-mint-dev-teststorage/demo/refFlat.chr21.txt",
+  "WrapperSs2RsemSingleSample.rrna_interval": "gs://broad-dsde-mint-dev-teststorage/demo/hg19.chr21.rRNA.interval_list",
+  "WrapperSs2RsemSingleSample.star_genome": "gs://broad-dsde-mint-dev-teststorage/demo/star.tar",
+  "WrapperSs2RsemSingleSample.rsem_genome": "gs://broad-dsde-mint-dev-teststorage/demo/rsem.tar",
+  "WrapperSs2RsemSingleSample.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
+  "WrapperSs2RsemSingleSample.format_map": "gs://broad-dsde-mint-dev-teststorage/format_map_example.json",
+  "WrapperSs2RsemSingleSample.submit_url": "http://api.ingest.staging.data.humancellatlas.org/",
+  "WrapperSs2RsemSingleSample.dss_url": "https://dss.staging.data.humancellatlas.org/v1",
+  "WrapperSs2RsemSingleSample.method": "Ss2RsemSingleSample",
+  "WrapperSs2RsemSingleSample.schema_version": "v3",
+  "WrapperSs2RsemSingleSample.run_type": "run",
+  "WrapperSs2RsemSingleSample.retry_seconds": 10,
+  "WrapperSs2RsemSingleSample.timeout_seconds": 600
```

Comment:
@dshiga Here I guess we need an environment parameter too: `"WrapperSs2RsemSingleSample.runtime_environment": "dev"`

Reply:
added